example/algorithms_evaluation/run.sh

```
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 <timeout (e.g., 10s, 1m, 1h)>"
+  exit 1
+fi
+
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)